xcode/Subconscious/Shared/Components/Detail/MemoEditorDetail.swift

```diff
@@ -613,7 +614,7 @@ struct MemoEditorDetailModel: ModelProtocol {
     var saveState = SaveState.saved
     
     /// Is editor in loading state?
-    var isLoading = true
+    var loadingState = LoadingState.loading
```

Comment:
Simple loading status support for the editor

Reply:
What happens if you edit a note while it is loading?